cmd/src/codeowners_create.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+func init() {
+	usage := `
+Examples:
+
+  Create a codeowners file for the repository "github.com/sourcegraph/sourcegraph":
+
+    	$ src codeowners create -repo='github.com/sourcegraph/sourcegraph' -f CODEOWNERS
+
+  Create a codeowners file for the repository "github.com/sourcegraph/sourcegraph" from stdin:
+
+    	$ src codeowners create -repo='github.com/sourcegraph/sourcegraph' -f -
+`
+
+	flagSet := flag.NewFlagSet("create", flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src codeowners %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		repoFlag = flagSet.String("repo", "", "The repository to attach the data to")
+		fileFlag = flagSet.String("f", "", "File path to read ownership information from (- for stdin)")
```

Comment:
could we make both `-file` and `-f` valid? (caught me out). if it's a big hassle nw, just wondering